client/state/jetpack-settings/modules/actions.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import i18n from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import {
+	JETPACK_MODULE_ACTIVATE,
+	JETPACK_MODULE_ACTIVATE_FAIL,
+	JETPACK_MODULE_ACTIVATE_SUCCESS
+} from 'state/action-types';
+import wp from 'lib/wp';
+
+export const activateModule = ( module_slug ) => {
+	return ( dispatch ) => {
+		dispatch( {
+			type: JETPACK_MODULE_ACTIVATE,
+			module: module_slug
+		} );
+		return new Promise( ( resolve, reject ) => {
```

Comment:
https://github.com/Automattic/wp-calypso/blob/master/client/lib/wpcom-undocumented/lib/undocumented.js#L81

Why are we defining a new 'activateJetpackModule' instead of using the existing 'jetpackModuleActivate' ?